### PR TITLE
Two new selfie-related filters

### PR DESCRIPTION
**Pull request by drzraf**
Added two selfie effects: blur and solid which render the selfie in a fixed color.
Implement a unified way to specify and implement selfie effects using a simpler but extendable syntax

### CODE REVIEW

**Comment by drzraf** [comment]
ping :) ?

**Comment by fangfufu** [comment]
It is not clear to me how you use the filters, and please reply with how your new filters look in this pull request.

**Comment by drzraf** [comment]
`--selfie=solid=250,0,0`

Screenshot from 2023-10-08 17-55-51


`--selfie=blur=94`

Screenshot from 2023-10-08 17-55-01


`--selfie=blur=94 --selfie=hologram`

Screenshot from 2023-10-08 17-54-45

`--selfie=solid=0,0,250` with a video background

Screenshot from 2023-10-08 18-29-21

File: fake.py [hunk 1]
```diff
@@ -388,6 +391,8 @@ def parse_args():
                         help="Foreground mask image path")
     parser.add_argument("--hologram", action="store_true",
                         help="Add a hologram effect")
+    parser.add_argument("--selfie", action='extend', nargs=1, default=[],
+                        help="Effects to apply to the self among hologram, solid=<N,N,N>, cmap=<name> or blur=<N>")
```
Comment (fangfufu) [inline fake.py:395]: Reword this, the description doesn't make much sense grammatically.

**Comment by fangfufu** [comment]
I prefer if you add the effect in a similar way as other examples.

**Comment by drzraf** [comment]
> I prefer if you add the effect in a similar way as other examples.

But as you can see, a very large number of effect is possible (and could be implemented in the future) since these are basically transformation of a frame. But there arguments are specific (and they could also be applied in a different order).
The point of `process_selfie_args` is to pave the way to better extensibility and readable code via consistent handling and ordering of argument (although it could be improved with a better long-term argument syntax a-la-ffmpeg) allowing for:
- simple transformation function
- unified code to trigger them

What would you envision as a better long-term solution to handle multiple filter (including those which could indistinctly apply to either the background or the foreground)?

**Comment by fangfufu** [comment]
> What would you envision as a better long-term solution to handle multiple filter (including those which could indistinctly apply to either the background or the foreground)?

That doesn't matter for now. Perhaps someone else will push in a better solution. 

It is not clear for me, from your submission, how to actually invoke the filter your added. You need to provide better documentation by updating README and the ``--help`` output. You still haven't addressed one of my review comment in your code.

**Comment by fangfufu** [comment]
A good example would be this: https://github.com/fangfufu/Linux-Fake-Background-Webcam/commit/89c80a3f66bb25eea13c78558c100fba7f33478a

**Pull request merged**